migration-engine/core/src/commands/mark_migration_rolled_back.rs

```diff
@@ -26,9 +26,11 @@ impl MigrationCommand for MarkMigrationRolledBackCommand {
 
     async fn execute<C: MigrationConnector>(input: &Self::Input, engine: &MigrationApi<C>) -> CoreResult<Self::Output> {
         // We should take a lock on the migrations table.
-
         let persistence = engine.connector().migration_persistence();
 
+        //todo the input currently does not take the migration directory as input. therefore no error atm, but I think the behaviour
+        // should be consistent between mark applied and mark rolled back
```

Comment:
Yes, consistency would be better — I think it's fine for now, though. This command doesn't use the local migration history at all.